golem/verificator/blender_verifier.py

```diff
@@ -106,25 +113,24 @@ def failure(exception):
             timeout=self.timeout)
 
         def error(exception):
-            logger.warning('Verification process exception %s', exception)
+            logger.warning(
+                f'Verification process exception. '
+                f'Subtask_id: {subtask_id}. Exception: {exception}')
             self.finished.errback(exception)
 
         def callback(*_):
             with open(os.path.join(dir_mapping.output, 'verdict.json'), 'r') \
                     as f:
                 verdict = json.load(f)
-
-            logger.info(
-                'Subtask %s verification verdict: %s',
-                subtask_info['subtask_id'],
-                verdict,
-            )
             if verdict['verdict']:
                 self.finished.callback(True)
             else:
                 self.finished.errback(
-                    Exception('Verification result negative', verdict))
+                    Exception('Verification result negative'))
 
+        logger.info(
+            f'Data for verification prepared. '
```

Comment:
if I get it right, it's more like:
```suggestion
            f' Starting docker thread for:  '
```